web/src/components/EpisodeViewer.vue

```diff
@@ -250,6 +256,34 @@ const checkProgress = async (force?: boolean) => {
     }
 }
 
+let playedOverHalfCounter = 0
+function onVideoProgress(
```

Comment:
I think we should point out in a comment that this is a first naive implementation and that you can just skip to end to get the streak.